address-model-lib/src/main/java/io/enmasse/address/model/Address.java

```diff
@@ -23,11 +23,12 @@
     private final String addressSpace;
     private final String type;
     private final String plan;
+    private final Optional<String> topic;
```

Comment:
This is just a java 'convention' nitpick. There seems to be an agreement that using Optional as a field in a data structure is a misuse of the API and should be avoided. Returning an Optional<> is however fine. Most of the code in this library is already following this convention.

The 'getTopic()' however should instead use `return Optional.ofNullable(topic)`